opt/scopeserver/dataserver/utils/labels.py

```diff
@@ -38,13 +38,10 @@ def label_annotation(loom: Loom, embedding: int, feature: str) -> List[FeatureLa
     at the barycentre of the cell cluster.
     """
     values = loom.get_meta_data_annotation_by_name(name=feature)["values"]
-    all_annotations = loom.get_ca_attr_by_name(name=feature)
-    colours = uniq((constant.BIG_COLOR_LIST[values.index(i)] for i in all_annotations))
-
-    annotations = uniq(loom.get_ca_attr_by_name(name=feature))
+    colours = [constant.BIG_COLOR_LIST[i % len(constant.BIG_COLOR_LIST)] for i in range(len(values))]
```

Comment:
```suggestion
    colours = toColours(range(len(values)))
```
Does this not break labels for clusters in the viewer?

Reply:
This doesn't touch clusters, they have their own function because of needing to deal with `-1` values